<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dicomifier::gui::SubjectsFrame</class>
 <widget class="QWidget" name="dicomifier::gui::SubjectsFrame">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>654</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>586</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit#dataDirectory
{
    margin-left: 5px;
    margin-right: 0px;
    margin-bottom: 25px;
    margin-top: 0px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="5" rowspan="2" alignment="Qt::AlignBottom">
    <widget class="QPushButton" name="browserButton">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>88</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#browserButton
{
	margin-right: 10px;
}</string>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="sortedByStudies">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Studies</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" alignment="Qt::AlignTop">
    <widget class="QLabel" name="dataDirectoryTitle">
     <property name="text">
      <string>Data Directory:</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="filtersTitle">
     <property name="text">
      <string>Filters:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="sortedTitle">
     <property name="text">
      <string>Sorted by:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="filtersName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="sortedBySubjects">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Subjects</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QCheckBox" name="selectAllCheckBox">
     <property name="styleSheet">
      <string notr="true">QCheckBox 
{
	margin-left: 25px;
 }
</string>
     </property>
     <property name="text">
      <string>Select All</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="fromDateTitle">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QLineEdit" name="dataDirectory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="6" rowspan="2">
    <widget class="QPushButton" name="refreshButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>88</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#refreshButton
{
	background-color: transparent;
	image: url(:/images/button_refresh) 0 0 0 0 stretch stretch;
	border: 1px solid #2299FF;
    border-radius: 10px;
    margin-top: 30px;
    margin-bottom: 10px;
    margin-left: 10px;
    margin-right: 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="toDateTitle">
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="7">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <zorder>browserButton</zorder>
    </widget>
   </item>
   <item row="4" column="5" colspan="2">
    <widget class="QDateTimeEdit" name="dateFilterEnd">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="5" colspan="2">
    <widget class="QDateTimeEdit" name="dateFilterBegin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
